src/ape_ethereum/transactions.py

```diff
@@ -84,6 +90,25 @@ class DynamicFeeTransaction(BaseTransaction):
     max_priority_fee: Optional[int] = Field(None, alias="maxPriorityFeePerGas")
     max_fee: Optional[int] = Field(None, alias="maxFeePerGas")
     type: Union[int, str, bytes] = Field(TransactionType.DYNAMIC.value)
+    access_list: List[AccessList] = Field(default_factory=list, alias="accessList")
```

Comment:
EIP-1559 transactions can have access list too, [see the spec](https://eips.ethereum.org/EIPS/eip-1559).